### PR TITLE
[tensorexpr] conv2d handle optional bias

### PR DESCRIPTION
**Pull request by IvanKobzarev**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #64751
* __->__ #64750
* #64741
* #64740

conv2d bias is optional. It will be ArgNone in processing of the graph.
This bias is prim::constant NoneType, so we do not know shape at the moment of constant binding.

This adding it as a constant zeros Tensor at the moment of graph processing => for that adding `std::vector<TensorExprKernel::ConstantDescr>& constants and std::vector<at::Tensor>& constant_tensors` to `computeOperandValue` as  it is not in `TensorExprKernel`

Differential Revision: [D30842101](https://our.internmc.facebook.com/intern/diff/D30842101)

### CODE REVIEW

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30842101).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30842101).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30842101).

File: torch/csrc/jit/tensorexpr/kernel.cpp [hunk 1]
```diff
@@ -2560,8 +2579,29 @@ Tensor TensorExprKernel::computeValue(const torch::jit::Value* v) {
...
     argInputs.push_back(toArg(inputs[0]));
   }
+
   auto outputType = findDtypeForValue(v->node()->output());
   std::vector<ExprHandle> outputShape = sizesForValue(v);
+  // handle ops optional arguments
+  switch (op) {
+    case aten::conv2d: {
+      // handle optional bias
+      if (c10::get_if<ArgNone>(&argInputs[2])) {
```
Comment (navahgar) [inline torch/csrc/jit/tensorexpr/kernel.cpp:2589]: Can we handle this without explicitly generating the bias tensor? Is this essential for the external call version?

The version of `conv2d` that NNC supports can handle cases without bias (https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/tensorexpr/operators/conv2d.h#L19), however, the caller is not doing the right thing which needs to be fixed.

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30842101).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30842101).

File: torch/csrc/jit/tensorexpr/kernel.cpp (hunk 1 above)
Comment (navahgar) [reply torch/csrc/jit/tensorexpr/kernel.cpp:2564]: Thanks for adding a TODO.

**Review by navahgar — APPROVED** [review]

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30842101).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30842101).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30842101).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30842101).

**Pull request closed without merging**